recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
```

Comment:
Either we clone the repo or we use the workaround @kalefranz [suggested]( https://github.com/conda-forge/staged-recipes/pull/656#issuecomment-229707113 ). Otherwise we cannot run `python setup.py install` as the version is undefined. Personally as this is how it is suppose to work I just went this way before knowing about the workaround. Setting the depth to one so we are not cloning unnecessarily. Welcome to discuss this though.